roles/olm/defaults/main.yaml

```diff
@@ -1,3 +1,5 @@
 ---
 operator_lifecycle_manager_remove: false
 operator_lifecycle_manager_install: true
+olm_operator_image: "{{ l_osm_registry_url | regex_replace('${component}' | regex_escape, 'operator-lifecycle-manager') }}"
+olm_catalog_operator_image: "{{ l_osm_registry_url | regex_replace('${component}' | regex_escape, 'operator-lifecycle-manager') }}"
```

Comment:
Will these always be the same image? Looks like prior to this PR they were different, not sure if this is a copy/paste error or not.
quay.io/coreos/olm versus quay.io/coreos/catalog

Reply:
Also, l_osm_registry_url will be 'docker.io/openshift/origin-${component}:${version}' for origin. Will that work for origin images too?
If not then we need something like this where we just rely on the replacement being a noop.

```
olm_registry_url_dict:
  origin: 'quay.io/coreos/operator-livecycle-manager'
  openshift-enterprise: "{{ l_osm_registry_url }}"
olm_operator_image: "{{ olm_registry_url_dict[openshift_deployment_type] | regex_replace('${component}' | regex_escape, 'operator-lifecycle-manager') }}"
olm_catalog_operator_image: "{{ olm_registry_url_dict[openshift_deployment_type] | regex_replace('${component}' | regex_escape, 'operator-lifecycle-manager') }}"
```

Reply:
For simplicity for now, we just threw both binaries into one image. I'll update with the quay urls for origin, though

Reply:
@sdodson Updated the image vars.

Is there a standard practice for how/when to tag origin images?

Reply:
Not that I know, we should probably establish a norm. In the past we'd just assumed everything was in docker.io/openshift.